flatpak/com.editor.gamma.json

```diff
@@ -1,41 +0,0 @@
-{
-    "app-id": "com.editor.gamma",
-    "runtime": "org.gnome.Platform",
-    "runtime-version": "3.34",
-    "sdk": "org.gnome.Sdk",
-    "command": "/app/bin/gamma",
-    "finish-args": [
-    	"--share=network",
-    	"--socket=x11",
-		"--share=ipc",
-		"--socket=session-bus",
-		"--filesystem=host",
-		"--own-name=com.editor.gamma"
-    ],
-    "modules": [
-    	{
-            "name": "gtksourceview",
-            "config-opts": [ "--disable-Werror" ],
-            "sources": [
-            {
-                "type": "archive",
-                "url": "https://download.gnome.org/sources/gtksourceview/4.0/gtksourceview-4.0.0.tar.xz",
-                "sha256": "6e5c7a28a7fa456a89f289f0659c57100cc09fe692db6abfcc23f7ade5d5b32a"
-            }
-            ]
-        },
-        {
-            "name": "gamma",
-            "buildsystem": "simple",
-            "build-commands": [
-            	"./flatpak/flatpak_setup.sh"
-            ],
-            "sources": [
-                {
```

Comment:
I don't think you want to remove the flatpak manifest. You just need to use a git type and point the url to your repository. Please make sure to point it to a specific tag/commit as Flathub builds have to be reproducible and point to a stable release

Reply:
Ok, I will try that.

Thanks

Reply:
Is there any way to have git url for a specific branch?

Reply:
no worries, I got the solution.

```
"type": "git",
                    "url": "https://gitlab.com/hamadmarri/gamma-text-editor",
                    "branch": "flatpak"
```

Reply:
No but you can use the branch & commit attribute.